Expensify/App
# Update copy to refer to new URL

### Details


### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/170729

### Tests
1. Sign into the app
2. Go to settings
3. Go to reset password
4. Verify the copy mentions "new.expensify.com"
5. Check the title of the apps (will be different in each platform) and see that they contain "new.expensify.com" now

### QA Steps
Same as tests

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/1228807/125853711-2eab9193-95e3-4781-9802-2ddab668184b.png)

![image](https://user-images.githubusercontent.com/1228807/125853698-88b33aad-a027-4a0b-ba42-6ffc5e5819f0.png)


#### Mobile Web
![image](https://user-images.githubusercontent.com/1228807/125853725-8ba44aea-6f05-4887-a7f8-70bc4553fde0.png)


#### Desktop
![image](https://user-images.githubusercontent.com/1228807/125853826-a6458050-b039-495b-952f-af9f8df76398.png)
![image](https://user-images.githubusercontent.com/1228807/125853865-9794d4ce-7db1-41bd-bf8c-57d9e6ca9d48.png)


#### iOS
![image](https://user-images.githubusercontent.com/1228807/125859034-df15dd66-838e-4a02-94e2-4b0bacae838e.png)


#### Android
![image](https://user-images.githubusercontent.com/1228807/125860529-a9518abe-833c-45fa-8804-161e4e32ed9b.png)
